flixel/FlxSprite.hx

```diff
@@ -636,7 +646,7 @@ class FlxSprite extends FlxObject
 				}
 				
 				_point.copyToFlash(_flashPoint);
-				camera.copyPixels(_frame, framePixels, _flashRect, _flashPoint, cr, cg, cb, alpha, blend, antialiasing);
+				camera.copyPixels(_frame, framePixels, _flashRect, _flashPoint, cr, cg, cb, alpha, croff, cgoff, cboff, caoff, blend, antialiasing);
```

Comment:
Man, 8 parameters for color multipliers / offsets. I feel like passing around a `ColorTransform` instance would be much nicer.